src/components/organisms/PageHeader/CloudSync/CloudSync.tsx

```diff
@@ -0,0 +1,217 @@
+import {useCallback} from 'react';
+
+import {Button, Dropdown, Spin, Tooltip} from 'antd';
+
+import {CheckCircleFilled, CloseCircleFilled, InfoCircleFilled, LoadingOutlined} from '@ant-design/icons';
+
+import styled from 'styled-components';
+
+import {TOOLTIP_DELAY} from '@constants/constants';
+
+import {useCloudPolicy, useCloudUser} from '@redux/validation/validation.hooks';
+
+import CloudIcon from '@assets/CloudIcon.svg';
+import CloudIconWhite from '@assets/CloudIconWhite.svg';
+import CloudSynced from '@assets/CloudSynced.png';
+import CloudUnsynced from '@assets/CloudUnsynced.png';
+
+import {Colors} from '@monokle/components';
+import {openUrlInExternalBrowser} from '@shared/utils';
+
+const LoadingIcon = <LoadingOutlined style={{fontSize: 24}} spin />;
+
+const CloudSync = () => {
+  const {connect, cloudUser, isConnecting, isInitializing} = useCloudUser();
+  const {cloudPolicy, projectInfo, policyInfo} = useCloudPolicy();
+
+  const dropdownRender = useCallback(() => {
+    if (isInitializing) {
+      return (
+        <DropdownContent>
+          <Spin indicator={LoadingIcon} />
+          Initializing...
+        </DropdownContent>
+      );
+    }
+
+    if (cloudUser) {
+      return (
+        <DropdownContent>
+          <Image src={CloudSynced} />
+          <Item>
+            <span>
+              <GreenCircle /> <GreenSpan>Connected</GreenSpan> <GraySpan>to Monokle Cloud</GraySpan>
+            </span>
+          </Item>
+          <Item>
+            <Title>
+              <GreenCircle /> E-mail
+            </Title>
+            <Value>{cloudUser.email}</Value>
+          </Item>
+          <Item>
+            <Title>{projectInfo?.name ? <GreenCircle /> : <RedCircle />} Cloud project</Title>
+            <Value>
+              {projectInfo?.name ? (
+                <>
+                  <Link onClick={() => openUrlInExternalBrowser(projectInfo.link)}>{projectInfo.name}</Link>
+                  <Info description="The Monokle Cloud project manages this repository." />
+                </>
+              ) : (
+                <>
+                  <span>Not found</span>
+                  <Info description="No Monokle Cloud Project that manages this repository was found." />
+                </>
+              )}
+            </Value>
+          </Item>
+          <Item>
+            <Title>{cloudPolicy ? <GreenCircle /> : <RedCircle />} Policy</Title>
+            <Value>
+              {cloudPolicy ? (
+                <>
+                  <Link onClick={() => openUrlInExternalBrowser(policyInfo?.link)}>View project policy</Link>
+                  <Info description="A Policy is configured for this repository in Monokle Cloud. Your local Policy is in-sync and cannot be changed." />
+                </>
+              ) : (
+                <>
+                  <span>Not found</span>
+                  <Info description="Couldn't find any Policy configured for this repository." />
```

Comment:
Can we add a [Configure Policy] link that goes to the policy builder?

Reply:
I can't add this easily right now, we can try it in a follow up after I talk to someone from Cloud